pkgs/development/python-modules/pylibgen/default.nix

```diff
@@ -0,0 +1,26 @@
+{ python3Packages, fetchPypi , python , stdenv }:
```

Comment:
remove python3Packages because this is called from the Python package set

Reply:
I assume I have to call with `pythonPackages` then?